tests/run-custom-args/tasty-inspector/tastyPaths.check

```diff
@@ -1,2 +1,2 @@
-List(/tastyPaths/I8163.class)
+List(/tastyPaths/I8163.tasty)
```

Comment:
does this count as an API change?

Reply:
Good catch. This falls into undefined behavior in the documentation. Therefore we can change it.

This comes from a call to `quotes.reflect.SourceFile.current`. In the inspector, we explicitly state that this method will not work 

https://github.com/lampepfl/dotty/blob/c62909009506171b03e092c4a391483b044d3b49/scaladoc/src/scala/tasty/inspector/Inspector.scala#L27